src/Symfony/Bridge/Doctrine/CacheWarmer/ProxyCacheWarmer.php

```diff
@@ -34,7 +34,7 @@ public function __construct(ManagerRegistry $registry)
     /**
      * This cache warmer is not optional, without proxies fatal error occurs!
      *
-     * @return false
+     * @return bool
```

Comment:
false is not allowed as a standalone type

Reply:
But it is allowed as a PHPdoc type: https://docs.phpdoc.org/3.0/guide/references/phpdoc/types.html

And also recognized by static analysis tools as such. Should we keep `@return false` and detect this as `: bool` in the class loader?

Reply:
`false` is allowed, but just not as a standalone type: it should always be part of a union